pairModule/fileHandling.py

```diff
@@ -13,35 +13,35 @@ def getPeople(team):
     return people
 
 
-def getPreviousPairs(team):
+def getPreviousPairObjs(team):
     with open(f"__pairfiles__/{team}/previous_pears.json") as f:
-        previousPairSets = json.load(f)
-    return previousPairSets
+        previousPairObjs = json.load(f)
+    return previousPairObjs
 
-def getMostRecentPairs(team):
-    pairs = getPreviousPairs(team)
-    if len(pairs) != 0:
-        return pairs[0]
+
+def getMostRecentPairObj(team):
+    pairObjs = getPreviousPairObjs(team)
+    if len(pairObjs) != 0:
+        return pairObjs[0]
     else:
         return []
 
 
-def savePairings(team, pairings, unpaired):
-    with open(f"__pairfiles__/{team}/previous_pears.json") as f:
-        previousPairSets = json.load(f)
-    previousPairSets.insert(0, {
-        **{
-            p1: p2
-            for p1, p2 in pairings
-        },
-        **{
-            person: 0
-            for person in unpaired
-        }
+def savePairings(team, name, pairings, excluded, unpaired, location, *, overwrite_most_recent=False):
```

Comment:
Not a specific code change but just wondered what the * argument means and what it's used for?

Reply:
* means that the arguments after that * cannot be passed as positional arguments, they have to be passed as keyword arguments

Reply:
`f(5, b=7)` then the first one is a normal arg, the second one is a kwarg